src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -13044,6 +13044,45 @@ System.Object
 
         End Sub
 
+		<Fact()>
+    	Public Sub Constraints()
```

Comment:
Please also add corresponding tests to the C# tests you added (`PointerTypes` and `LongTuplePointerTypes`)